lib/cli/src/Cardano/CLI.hs

```diff
@@ -1022,12 +1022,9 @@ newtype Port (tag :: Symbol) = Port { getPort :: Int }
 -- TCP port ranges from [[-1;65535]] \ {0}
 -- However, ports in [[-1; 1023]] \ {0} are well-known ports reserved
 -- and only "bindable" through root privileges.
---
--- Ports above 49151 are also reserved for ephemeral connections. This
--- leaves us with a valid range of [[1024;49151]] for TCP ports.
 instance Bounded (Port tag) where
     minBound = Port 1024
-    maxBound = Port 49151
+    maxBound = Port 65535
```

Comment:
Seems totally reasonable to use the full range of ports. If the OS selects a port then it's a good port to use.

The minBound could also be 0, because a. the wallet could be connecting to a node listening on any port, and b. in some contexts such as a docker container, the wallet may well be running as "root" and thus be able to listen on any available port. But I'm not sure how bad the exception message is when the port can't be used, so maybe leave it.